src/components/topic/TopicService.js

```diff
@@ -71,10 +71,15 @@ goog.require('ga_permalink');
         });
 
         // Returns a promise that is resolved when topics are loaded
-        this.getTopics = function() {
+        this.loadConfig = function() {
           return topicsP;
```

Comment:
It will be impossible if wereturn the promise of the `then` function of the service not the promise of the `$http.get` so in th $q.all `then `calls the topics and topic are setted.

So there is a problem yes :)